### PR TITLE
feat: add dbt server integration

### PR DESCRIPTION
**Pull request by owlas**
Closes:

### CODE REVIEW

File: packages/backend/src/models/ProjectModel/ProjectModel.ts [hunk 1]
```diff
@@ -621,4 +623,64 @@ export class ProjectModel {
...
+                'projects.project_id',
+                'dbt_cloud_integrations.project_id',
+            )
+            .where('project_uuid', projectUuid);
+        if (row === undefined) {
+            return undefined;
+        }
+        return {
+            metricsJobId: row.metrics_job_id,
+            serviceToken: row.service_token,
```
Comment (ZeRego) [inline packages/backend/src/models/ProjectModel/ProjectModel.ts:658]: don't we need to decrypt the token ?

File: packages/frontend/src/components/ProjectConnection/WarehouseForms/SnowflakeForm.tsx [hunk 2]
```diff
@@ -155,6 +155,19 @@ const SnowflakeForm: FC<{
                     }
                     disabled={disabled}
                 />
+                <Input
```
Comment (ZeRego) [inline packages/frontend/src/components/ProjectConnection/WarehouseForms/SnowflakeForm.tsx:158]: Let's move this to advanced section ?

**Review by ZeRego — APPROVED** [review]

File: packages/backend/src/database/migrations/20220915192840_add_dbt_cloud_integrations_table.ts [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        await knex.schema.createTable(
+            'dbt_cloud_integrations',
+            (tableBuilder) => {
+                tableBuilder
+                    .integer('project_id')
+                    .primary()
+                    .references('project_id')
+                    .inTable('projects')
+                    .onDelete('CASCADE');
+                tableBuilder.binary('service_token').notNullable();
```
Comment (ZeRego) [inline packages/backend/src/database/migrations/20220915192840_add_dbt_cloud_integrations_table.ts:14]: What is the advantage of using binary over text ?

**Comment by owlas** [comment]
rebased + adding docs

**Pull request merged**